vips/vip-converter/vip-converter-testnet/commands.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  ACM,
+  BTCBPrimeConverterTokenOuts,
+  BaseAssets,
+  CONVERTER_NETWORK,
+  ETHPrimeConverterTokenOuts,
+  GUARDIAN,
+  RiskFundConverterTokenOuts,
+  TimelocksArray,
+  USDCPrimeConverterTokenOuts,
+  USDTPrimeConverterTokenOuts,
+  XVSVaultConverterTokenOuts,
+  converters,
+} from "./Addresses";
+
+type IncentiveAndAccessibility = [number, number];
+
+interface AcceptOwnership {
+  target: string;
+  signature: string;
+  params: [];
+}
+
+interface ConverterCommand {
+  target: string;
+  signature: string;
+  params: [string];
+}
+
+interface CallPermission {
+  target: string;
+  signature: string;
+  params: [string, string, string];
+}
+
+export const grant = (target: string, signature: string, caller: string): CallPermission => {
+  const config: CallPermission = {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [target, signature, caller],
+  };
+
+  return config;
+};
+
+function getIncentiveAndAccessibility(tokenIn: string, tokenOut: string): IncentiveAndAccessibility {
+  const validTokenIns = [BaseAssets[2], BaseAssets[3], BaseAssets[4], BaseAssets[5]];
+
+  if (validTokenIns.includes(tokenIn) && tokenOut === BaseAssets[0]) {
+    return [0, 2]; // ONLY_FOR_CONVERTERS
+  } else {
+    return [0, 1]; // ALL
+  }
+}
+
+function generateAcceptOwnershipCommands(ConvertersArray: string[]): AcceptOwnership[] {
+  const acceptOwnershipCommandsArray: AcceptOwnership[] = [];
+
+  for (const converter of ConvertersArray) {
+    const config: AcceptOwnership = {
+      target: converter,
+      signature: "acceptOwnership()",
+      params: [],
+    };
+
+    acceptOwnershipCommandsArray.push(config);
+  }
+
+  return acceptOwnershipCommandsArray;
+}
+
+function generateConverterCommands(ConvertersArray: string[]): ConverterCommand[] {
+  const commandsArray: ConverterCommand[] = [];
+
+  for (const converter of ConvertersArray) {
+    // Add AddConverterNetwork command
+    const addConverterNetworkConfig: ConverterCommand = {
+      target: converter,
+      signature: "setConverterNetwork(address)",
+      params: [CONVERTER_NETWORK],
+    };
+
+    // Add AddTokenConverter command
+    const addTokenConverterConfig: ConverterCommand = {
+      target: CONVERTER_NETWORK,
+      signature: "addTokenConverter(address)",
+      params: [converter],
+    };
+
+    commandsArray.push(addConverterNetworkConfig);
+    commandsArray.push(addTokenConverterConfig);
+  }
+
+  return commandsArray;
+}
+
+function generateCallPermissionCommands(ConvertersArray: string[]): CallPermission[] {
+  const callPermissionCommandsArray: CallPermission[] = [];
+
+  for (const converter of ConvertersArray) {
+    for (let i = 0; i < 3; i++) {
```

Comment:
I would avoid hardcoded numbers:
```suggestion
    for (const timelock of TimelocksArray) {
```

Reply:
Done https://github.com/VenusProtocol/vips/pull/49/commits/9eb73c07ce3c22d0e06dac9f5ba757469d673f1c